here-positioning-android/README

```diff
@@ -0,0 +1,8 @@
+This tutorial application does not define HERE credentials for following items:
```

Comment:
Please remove this file in favor of adding a `app/libs/README.md` to align with existing examples (simply copy the file from another example).

Reply:
Funny, because I copied this file from one of the projects. See https://github.com/heremaps/here-android-sdk-examples/tree/master/advanced-navigation-android/app/libs